integration/spring-pulsar/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+spring.pulsar.client.service-url=pulsar://localhost:${PULSAR_PORT_6650:6650}
```

Comment:
@sobychacko we need to use `${PULSAR_HOST:localhost}` rather than `localhost` as per https://github.com/spring-projects/spring-aot-smoke-tests/pull/151#issuecomment-1323280946